gulpfile.js

```diff
@@ -161,13 +163,17 @@ function cordovaCompile() {
   // Use flag -UseModernBuildSystem=0 as a workaround for Xcode 10 compatibility until upgrading to
   // cordova-ios@5.0.0. See https://github.com/apache/cordova-ios/issues/404.
   const compileArgs = platform === 'ios' ? '--device --buildFlag="-UseModernBuildSystem=0"' : '';
-  const releaseArgs = isRelease ? platform === 'android' ?
-                                  `--release -- --keystore=${gutil.env.KEYSTORE} ` +
-              `--storePassword=${gutil.env.STOREPASS} --alias=${gutil.env.KEYALIAS} ` +
-              `--password=${gutil.env.KEYPASS}` :
-                                  '--release' :
-                                  '';
-  return runCommand(`cordova compile ${platform} ${compileArgs} ${releaseArgs} -- ${platformArgs}`);
+  const releaseArgs = [];
+  if (isRelease) {
+    releaseArgs.push('--release');
+    if (platform === 'android') {
+      releaseArgs.push(
```

Comment:
Let's use a string template to avoid the array to string conversion.

Reply:
Done in 15cbe65